### DISCUSSION TITLE
is it possible to open new windows in specific zone?

### DISCUSSION THREAD

**Discussion by SoyDiego** [gerritdevriese/kzones#83] [Q&A] [answered]
Hi @gerritdevriese!
First I want to say congratulations for the excellent project. I love kzones and I'm new using it.

I have created my zones and my only doubt here if I can open all my new windows directly in my right column. Now when I open new windows, always is opened centered and I have left/right column.

Thanks for your time!

**Comment by gerritdevriese** [comment, marked as answer]
Hi, this is now possible on the main branch (not released yet)  by adding your window classes to an `applications` key inside your layouts json. Example: https://github.com/gerritdevriese/kzones/blob/main/examples/advanced.json

**Comment by SoyDiego** [comment]
oh sorry I didn't received a notification about my topic and now searching again the possibility to do that, I saw you did a new feature adding this option!
I will try it right now hahaha, and thanks for hear questions and doubt like this. You are doing a great job with kzones! =)

Edit, sorry. Now I have read is not released yet.
Do you know when you will release? Thanks again @gerritdevriese !!
